Remotion/ObjectBinding/Web/UI/Controls/Validation/Factories/ValidationBusinessObjectBoundEditableWebControlValidatorFactory.cs

```diff
@@ -112,63 +112,60 @@ public IEnumerable<BaseValidator> CreateValidators (IBocTextValue control, bool
     {
       ArgumentUtility.CheckNotNull("control", control);
 
```

Comment:
Need to also upate ValidationBocReferenceDataSourceValidatorFactory and ValidationUserControlBindingValidatorFactory

Please check for unit tests. 

and fyi: there are no webtests for this type.